virtManager/delete.py

```diff
@@ -73,8 +73,6 @@ def _delete_vm(self, vm):
 
     def _delete_disks(self, vm, paths, conn, meter):
         storage_errors = []
-        if not paths:
-            return
```

Comment:
Please submit this as a separate patch. You can subject it: 'delete: remove redundant code'